spec/requests/access_site_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe 'Access site', type: :request do
```

Comment:
in Rspec.describe you should have the class that you are testing, not a description.